src/main/java/it/pagopa/swclient/mil/controller/model/CommonHeader.java

```diff
@@ -0,0 +1,16 @@
+package it.pagopa.swclient.mil.controller.model;
+
+import it.pagopa.swclient.mil.util.ErrorCodes;
+import it.pagopa.swclient.mil.util.RegexPatterns;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.ws.rs.HeaderParam;
+
+public record CommonHeader(@HeaderParam("RequestId")
```

Comment:
At the moment this record has usages only in test classes. Do we plan to use it or it can be removed (or moved in test package)?

Reply:
Yes, we don't need this. I removed it in latest commit